spring-web/src/main/java/org/springframework/http/RequestEntity.java

```diff
@@ -138,22 +148,38 @@ public RequestEntity(@Nullable T body, @Nullable MultiValueMap<String, String> h
 
 	/**
 	 * Constructor with method, URL, headers, body and type.
-	 * @param body the body
+	 * @param body    the body
 	 * @param headers the headers
-	 * @param method the method
-	 * @param url the URL
-	 * @param type the type used for generic type resolution
+	 * @param method  the method
+	 * @param url     the URL
+	 * @param type    the type used for generic type resolution
 	 * @since 4.3
 	 */
 	public RequestEntity(@Nullable T body, @Nullable MultiValueMap<String, String> headers,
-			@Nullable HttpMethod method, URI url, @Nullable Type type) {
+						 @Nullable HttpMethod method, URI url, @Nullable Type type) {
 
 		super(body, headers);
 		this.method = method;
 		this.url = url;
 		this.type = type;
 	}
 
+	/**
+	 * Private Constructor with method, URL, UriTemplate and varargs urivariables but without body nor headers.
+	 * @param method       the method
+	 * @param url          the URL
+	 * @param uriTemplate  the UriTemplate
+	 * @param uriVariables the uriVariables
+	 */
+	private RequestEntity(MultiValueMap<String, String> headers,HttpMethod method, URI url, UriTemplate uriTemplate, Object... uriVariables) {
+		super(null, headers);
+		this.method = method;
+		this.url = url;
+		this.type = null;
+		this.uriTemplate = uriTemplate;
+		this.uriVariables = uriVariables;
+	}
```

Comment:
Here you'll find our [Source Code Style](https://github.com/spring-projects/spring-framework/wiki/Code-Style) guidelines. Specifically the section on line wrapping (aim for 90, no more than 120, but only 80 for Javadoc).

Reply:
done